[QA] [Testnet] Not able to claim asset on Base Goerli

## Describe the current behavior in detail
Not able to claim WETH on Base when sending 0.01 ETH from Ethereum (Metamask) to Base (Metamask).

Tested from https://wormhole-connect.netlify.app/ [Testnet] with macOS Monterey 12.6.7 and Chrome Version 115.0.5790.170 (Official Build) (x86_64).

Send from tx: [Etherscan](https://goerli.etherscan.io/tx/0x527eaf569bfdce0cba8b6f6059f46f5a8f55707be4f8a31403dbedad99a9ece3)

Error in Console:
```
Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"from":"0x0a370E7c5F7F944d451B48bc151a7BEC454bc185","gasLimit":{"type":"BigNumber","hex":"0x0186a0"},"to":"0xA31aa3FDb7aF7Db93d18DDA4e19F811342EDF780","data":"0xc687851900000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000100010000000001002d18f92c45cee29558c989972fddbc785991a494f6c3bbe5507a18c177b181c73787edc949bfc069e4f406b2a1d4e777db21226857913948a220c792e6f6d0980164d0a30c09a000000002000000000000000000000000f890982f9310df57d00f659cf4fd87e65aded8d70000000000028276010100000000000000000000000000000000000000000000000000000000000f4240000000000000000000000000b4fbf271143f4fbf7b91a5ded31805e42b2208d600020000000000000000000000000a370e7c5f7f944d451b48bc151a7bec454bc185001e0000000000000000000000000000000000000000000000000000000000000000","accessList":null}, error={"code":-32603,"message":"Internal JSON-RPC error.","data":{"code":-32000,"message":"execution reverted"},"stack":"{\n  \"code\": -32603,\n  \"message\": \"Internal JSON-RPC error.\",\n  \"data\": {\n    \"code\": -32000,\n    \"message\": \"execution reverted\"\n  },\n  \"stack\": \"Error: Internal JSON-RPC error.\\n    at new n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:33894)\\n    at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:36834)\\n    at Object.internal (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:37444)\\n    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:44268)\\n    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:45267\\n    at async chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-3.js:12:54199\"\n}\n  at new n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:33894)\n  at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:36834)\n  at Object.internal (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:37444)\n  at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:44268)\n  at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:45267\n  at async chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-3.js:12:54199"}, code=CALL_EXCEPTION, version=providers/5.7.2)
    at l.makeError (index.ts:269:28)
    at l.throwError (index.ts:281:20)
    at g (json-rpc-provider.ts:66:16)
    at u.<anonymous> (json-rpc-provider.ts:642:20)
    at Generator.throw (<anonymous>)
    at s (formatter.ts:523:1)
```

Steps:
1. Connect Metamask in 'From' and 'To' sections
2. Send 0.01 ETH from 'Goerli' to 'Base Goerli'
3. Click 'Claim' button on Redeem 

![image](https://github.com/wormhole-foundation/wormhole-connect/assets/10746654/be4b7203-35a9-428d-8afb-aa51322c1d36)

![image](https://github.com/wormhole-foundation/wormhole-connect/assets/10746654/daa6298e-64f6-4b12-9a2a-3b8576a9308e)
 
## What is the expected/desired behavior
Asset is successfully sent and received.

Also reproducible when sending SOL from Solana (Phantom) to Base Goerli (Metamask).
Sending from tx: [Solana Explorer](https://explorer.solana.com/tx/qt4BsDMTtN2N9AKhACrP4bUWSvYCPXfSST21iFGJiANj97dke3cSXRB144oicAhysuHWDWDxy2RjYRPTEyhzsFd?cluster=devnet), tx hash `qt4BsDMTtN2N9AKhACrP4bUWSvYCPXfSST21iFGJiANj97dke3cSXRB144oicAhysuHWDWDxy2RjYRPTEyhzsFd`
Checked from https://base-relayer--wormhole-connect.netlify.app/.
Error in Console:
```
SendTo.tsx:295 Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"from":"0x0a370E7c5F7F944d451B48bc151a7BEC454bc185","gasLimit":{"type":"BigNumber","hex":"0x0186a0"},"to":"0xA31aa3FDb7aF7Db93d18DDA4e19F811342EDF780","data":"0xc68785190000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000010001000000000100ce1dd71bef6b62c400ff102547e2afd518e4b85a45c43888ad96df5fc38ef14f5a61c36a0e046f522c3ade05fe347fa240522e5c4352b490f5f50ed2a15670e50064d34d5d0001569400013b26409f8aaded3f5ddca184695aa6a0fa829b0c85caf84856324896d214ca9800000000000060ae200100000000000000000000000000000000000000000000000000000000000f4240069b8857feab8184fb687f634618c035dac439dc1aeb3b5598a0f0000000000100010000000000000000000000000a370e7c5f7f944d451b48bc151a7bec454bc185001e0000000000000000000000000000000000000000000000000000000000000000","accessList":null}, error={"code":-32603,"message":"Internal JSON-RPC error.","data":{"code":-32000,"message":"execution reverted"},"stack":"{\n  \"code\": -32603,\n  \"message\": \"Internal JSON-RPC error.\",\n  \"data\": {\n    \"code\": -32000,\n    \"message\": \"execution reverted\"\n  },\n  \"stack\": \"Error: Internal JSON-RPC error.\\n    at new n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:33894)\\n    at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:36834)\\n    at Object.internal (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:37444)\\n    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:44268)\\n    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:45267\\n    at async chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-3.js:12:54199\"\n}\n  at new n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:33894)\n  at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:36834)\n  at Object.internal (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-2.js:16:37444)\n  at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:44268)\n  at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:45267\n  at async chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-3.js:12:54199"}, code=CALL_EXCEPTION, version=providers/5.7.2)
    at l.makeError (index.ts:269:28)
    at l.throwError (index.ts:281:20)
    at g (json-rpc-provider.ts:66:16)
    at u.<anonymous> (json-rpc-provider.ts:642:20)
    at Generator.throw (<anonymous>)
    at s (formatter.ts:523:1)
```

--- a/wormhole-connect/src/utils/routes/bridge.ts
+++ b/wormhole-connect/src/utils/routes/bridge.ts
@@ -166,12 +166,7 @@ export class BridgeRoute extends BaseRoute {
       );
     }
 
-    const tx = await wh.redeem(
-      destChain,
-      messageInfo.rawVaa,
-      { gasLimit: 250000 },
-      payer,
-    );
+    const tx = await wh.redeem(destChain, messageInfo.rawVaa, undefined, payer);
     const txId = await signAndSendTransaction(
       destChainName,
       tx,

